app/main.py

```diff
@@ -1,3 +1,56 @@
+from typing import Union
+
+
 class Distance:
     # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Union[float, "Distance"]) -> "Distance":
```

Comment:
```suggestion
    def __add__(self, other: Union[float, int, Distance]) -> Distance:
```

Reply:
fix everywhere

Reply:
> fix everywhere

if i delete quotes i get the next error: "NameError: name 'Distance' is not defined"